administrative-sdk/organisation.js

```diff
@@ -89,56 +46,38 @@ class Organisation {
    * @param {Sdk~getCallback} [cb] The callback that handles the response.
    * @param {Sdk~getErrorCallback} [ecb] The callback that handles the error response.
    */
-  static getOrganisation(connection, organisationId, cb, ecb) {
-    var _cb = function(data) {
-      var organisation = new Organisation(data.id, data.name);
-      organisation.created = new Date(data.created);
-      organisation.updated = new Date(data.updated);
-      if (cb) {
-        cb(organisation);
-      }
-    };
+  static getOrganisation(connection, organisationId) {
     var url = connection.settings.apiUrl + '/organisations/' + organisationId;
-    connection._secureAjaxGet(url, _cb, ecb);
-  }
-
-  /**
-   * Callback used by listOrganisations.
-   *
-   * @callback Sdk~listCallback
-   * @param {its.Organisation[]} organisation Retrieved organisation domain model instances.
-   */
-  organisationListCallback(organisation) {}
 
-  /**
-   * Error callback used by listOrganisations.
-   *
-   * @callback Sdk~listErrorCallback
-   * @param {object[]} errors Array of errors.
-   */
-  organisationListErrorCallback(errors) {}
+    return connection._secureAjaxGet(url)
+      .then(data => {
+        var organisation = new Organisation(data.id, data.name);
+        organisation.created = new Date(data.created);
+        organisation.updated = new Date(data.updated);
+        return (organisation);
+      });
+  }
 
   /**
    * List all organisations in the organisation.
    *
    * @param {Sdk~listCallback} cb The callback that handles the response.
    * @param {Sdk~listErrorCallback} [ecb] The callback that handles the error response.
```

Comment:
Missing documentation update. Callbacks are no longer taken and a promise is returned.